### ISSUE TITLE
[BUG][Accessibility] Search results facets do not indicate state

### ISSUE THREAD

**Issue by spudthebud** [DOAJ/doaj#2167]
I'm sighted and certified with the JAWS screen reader. 

### **Fictitious User story:**
I am a blind researcher using JAWS. I am doing a search for a journal. After the search, I would like to filter by language.

1. Using JAWS, do a  journal search of DOAJ
2. Go to the Language facet in the search results. Notice that JAWS does not announce the collapsed or expanded state.

An example of the expected behaviour is [https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/)

### **Screenshots**

![image](https://user-images.githubusercontent.com/8799391/199661291-b4b10590-e847-4bdb-b7a6-496b21e51be6.png)

**Desktop (please complete the following information):**
 - OS: WIndows 11
 - Browser [e.g. chrome 106]
 - JAWS: 2022

**Comment by dommitchell** [comment]
Thank you for reporting this. We have duplicated the bug to our bug repo and I will update here once your ticket has been considered.

**Comment by dommitchell** [comment]
Hello, just to let you know that the changes you recommended to us have now gone live on our site. Thanks again for reporting this to us.
